airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3952,7 +3952,10 @@ components:
           type: string
           nullable: true
         priority_weight:
-          type: integer
+          anyOf:
+            - type: integer
+            - type: number
+              format: float
```

Comment:
Will this field actually return `int` value after applying migration?

Reply:
I added this because of backward compatibility.  
In the previous PRs, there were some discussions around backward compatibility. The thing is that some people say we are recommending the clear and backup the db before upgrade but it is not **a must**. 
If the user resets the DB, then it is not a problem. However, in case the user does not reset the DB then we need to be backward compatible because of the existing data.

Reply:
Fare enough. Thanks for the explanation!